library/built-in/SDXL/_cushySDXLLayout.tsx

```diff
@@ -5,9 +5,13 @@ import type { $CushySDXLUI } from './_cushySDXLSchema'
 
 import { observer } from 'mobx-react-lite'
 
+import { Button } from '../../../src/csuite/button/Button'
 import { ShellOptionalEnabledUI } from '../../../src/csuite/fields/optional/WidgetOptional'
+import { Frame } from '../../../src/csuite/frame/Frame'
 import { IkonOf } from '../../../src/csuite/icons/iconHelpers'
 import { InputNumberUI } from '../../../src/csuite/input-number/InputNumberUI'
+import { ResizableFrame } from '../../../src/csuite/resizableFrame/resizableFrameUI'
+import { PromptEditorUI } from '../../../src/prompt/__TEMP__'
```

Comment:
really need to make sure we properly see when we suddenly start to include cushy-codebase into bundled extensions; for now, it's not visible enough. I'm